contracts/pcv/lido/EthLidoPCVDeposit.sol

```diff
@@ -154,11 +172,11 @@ contract EthLidoPCVDeposit is PCVDeposit {
 
         // Compute the minimum accepted amount of ETH out of the trade, based
         // on the slippage settings.
-        Decimal.D256 memory maxSlippage = Decimal.ratio(
```

Comment:
Previously, we were assuming 1 ETH = 1stETH right. Now as it's evident there are peg issues we are using an oracle to factor in how much 1 stETH is worth

Reply:
yes, so for instance, if 1 stETH = 0.9 ETH, and there is a slippage tolerance of 10%, the withdraw will tolerate withdrawals down to 0.81 ETH per stETH

Reply:
Nice